x-pack/plugins/alerts/server/task_runner/task_runner.test.ts

```diff
@@ -545,10 +557,13 @@ describe('Task Runner', () => {
         Array [
           Object {
             "event": Object {
-              "action": "execute",
-              "outcome": "success",
+              "action": "resolved-instance",
             },
             "kibana": Object {
+              "alerting": Object {
+                "action_group_id": undefined,
```

Comment:
again, seems like `action_group_id` should not be `undefined` here, but ... for `resolved-instance`, I guess it's iffy

Reply:
We weren't passing it in (again), but also had to restructure things so we **could** pass it in.